### DISCUSSION TITLE
Error during updating LXCs

### DISCUSSION THREAD

**Discussion by WaW1190** [tteck/Proxmox#2289] [Help Needed] [answered]
I was trying to update several LXCs (created with the helper scripts) and got the following error:

[ERROR] in line 70: exit code 127: while executing command npm install --production &> /dev/null
(as an example from an UptimeKuma LXC).

I would appreciate any help for solving this issue.

**Comment by tteck** [comment, marked as answer]
fixed UptimeKuma https://github.com/tteck/Proxmox/commit/6e51ac0e69b7b4af72fa2c1b85ea70ebc4dfeb65

**Reply by tteck** [reply]
Let me know the others, and I'll fix them.

**Reply by WaW1190** [reply]
thanks for your quick response

Homepage also threw a similar error

**Reply by WaW1190** [reply]
Homepage still throws this error:

[ERROR] in line 59: exit code 100: while executing command apt-get install -y npm > /dev/null 2>&1

**Reply by tteck** [reply]
execute `apt-get install -y npm` in the console to show more details

**Reply by tteck** [reply]
Make sure you're using node v18 `node -v`

**Reply by WaW1190** [reply]
Now my disk was full - sorry, I did not check before the update.


Everything worked after resizing


Thank you so much, another donate is on its way

**Reply by tteck** [reply]
Thank you
